build.sbt

```diff
@@ -1,18 +1,18 @@
 
 inThisBuild(
-  (homepage := Some(url("https://github.com/gemini-hlsw/sbt-gsp"))) +: gspPublishSettings
+  (homepage := Some(url("https://github.com/gemini-hlsw/sbt-lucuam"))) +: lucumaPublishSettings
 )
 
 addSbtPlugin("de.heikoseeberger"         % "sbt-header"   % "5.6.0")
 addSbtPlugin("io.github.davidgregory084" % "sbt-tpolecat" % "0.1.13")
 addSbtPlugin("ch.epfl.scala"             % "sbt-scalafix" % "0.9.20")
 
-lazy val sbtGsp = (project in file("."))
-  .disablePlugins(GspPlugin)
+lazy val lucumaGsp = (project in file("."))
```

Comment:
Maybe it doesn't matter but `sbtLucuma` here I think?

Reply:
Fixed